lib/features/profile/data/models/profile_model.dart

```diff
@@ -0,0 +1,78 @@
+// import 'package:final_project/features/profile/domain/entities/profile.dart';
```

Comment:
Don't commit commented code to version control. Read on the benefits of Version Control.